src/HonzaBotner/Startup.cs

```diff
@@ -97,23 +107,53 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     c.SwaggerEndpoint("/swagger/v1/swagger.json", "HonzaBotner v1");
                     c.RoutePrefix = string.Empty;
                 });
+                app.UseHttpsRedirection();
+                app.UseHangfireDashboard();
             }
             else
             {
                 UpdateDatabase(app);
+                SetupDashboard(app);
                 app.UseReverseProxyHttpsEnforcer();
             }
 
+            StartRecurringJobs(app);
+
             app.UseHttpsRedirection();
+            app.UseHangfireServer();
             app.UseRouting();
             app.UseEndpoints(endpoints => { endpoints.MapControllers(); });
         }
 
+        private void StartRecurringJobs(IApplicationBuilder app)
+        {
+            using IServiceScope scope = app.ApplicationServices
+                .GetRequiredService<IServiceScopeFactory>()
+                .CreateScope();
+
+            // TODO: Make this extendable with other jobs
+            var job = scope.ServiceProvider.GetService<TriggerRemindersJobProvider>()!;
+
+            RecurringJob.AddOrUpdate(
+                job.Key,
+                () => job.Run(),
+                job.CronExpression
+            );
+        }
```

Comment:
This will potentialy cause issues with disposing of scope with running jobs. Consider using this: https://docs.hangfire.io/en/latest/background-methods/using-ioc-containers.html